packages/yoastseo/src/scoring/assessments/seo/ProductSKUAssessment.js

```diff
@@ -3,7 +3,6 @@ import AssessmentResult from "../../../values/AssessmentResult";
 import { merge } from "lodash-es";
 import { createAnchorOpeningTag } from "../../../helpers/shortlinker";
 import { __, sprintf } from "@wordpress/i18n";
```

Comment:
```suggestion
import { __, sprintf } from "@wordpress/i18n";

```

Reply:
There should be extra new line between import statement and other codes below

Reply:
'woo'ps